addon/src/model-definition.js

```diff
@@ -127,10 +138,7 @@ class ModelDefinition {
 
     // set the id, unless it was already set in opts
     if (!fixture.id && !opts.id) {
-      // Setting a flag to indicate that this is a generated an id,
-      // so it can be rolled back if the fixture throws an error.
-      fixture._generatedId = true;
```

Comment:
I don't think it's necessary to "rollback an id" if the make failed - and isn't something we can do with string ids. It a make failed, then that generated id just won't get used